db-service/lib/cqn2sql.js

```diff
@@ -423,21 +423,39 @@ class CQN2SQLRenderer {
   // STREAM Statement -------------------------------------------------
 
   STREAM(q) {
-    let { from, into, where, column, data } = q.STREAM
+    let { from, into, where, column, columns, data } = q.STREAM
     let x, sql
     // reading stream
     if (from) {
       sql = `SELECT`
-      if (!_empty((x = column))) sql += ` ${this.quote(x)}`
+      if (!_empty((x = column))) {
```

Comment:
The implementation looks increasingly complex → hard to understands and maintain?